hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/replication/ReplicationSupervisor.java

```diff
@@ -147,6 +150,17 @@ public void run() {
       final Long containerId = task.getContainerId();
       try {
         requestCounter.incrementAndGet();
+        if (task.getTimeoutMs() != 0) {
+          long msInQueue =
```

Comment:
Rather than calling Instance.now here, can we inject an instance of `MonotonicClock` and use it to get the time. This will make tests easier and avoid `sleep` calls in the tests. This is something we have used in other places in the code before.